lib/lita/adapters/slack/slack_channel.rb

```diff
@@ -18,7 +18,7 @@ def from_data(channel_data)
 
           # @api private
           def from_data_array(channels_data)
-            channels_data.map { |channel_data| from_data(channel_data) }
+            channels_data["channels"].map { |channel_data| from_data(channel_data) }
```

Comment:
Data from the `conversations.list` endpoint now comes wrapped in a `channels` array ([source](https://api.slack.com/methods/conversations.list#responses))